### ISSUE TITLE
Test using pidgeons.jl inference methods with EpiAware models

### ISSUE THREAD

**Issue by SamuelBrand1** [CDCgov/Rt-without-renewal#457]
The [`Pigeons.jl`](https://github.com/Julia-Tempering/Pigeons.jl) package exposes parallel tempering for `Turing` models. PT is considered an effective inference methodology for posterior distributions with complicated geometries, and the `Pigeons.jl` package looks very interesting.

Identifiability/complex posterior geometry problems abound in stats epidemiology so it would be good to look at pigeons.

**Comment by SamuelBrand1** [comment]
I think a good test case would be: https://cdcgov.github.io/Rt-without-renewal/dev/showcase/replications/mishra-2020/

As mentioned in the vignette, there is non-identifiability between the neg bin observation dispersion and the noise of the AR(2) process for $\log R_t$. It would be interesting to see how `Pigeons.jl` deals with that.

**Comment by seabbs** [comment]
I thought the most/more interesting element was being able to run across many cores at once?

**Comment by SamuelBrand1** [comment]
I don't think that the multi-core aspect is as exciting per se, its more that by having a lot of cores you can create a "communication" bridge between an easy to sample distribution and a hard to sample distribution via a large number of intermediate distributions. 

However, its still MH... just from theoretical considerations I expect it to be outperformed by NUTS on a few cores for not-so-hard distributions. Happy to see some counter evidence though.

**Comment by SamuelBrand1** [comment]
Full disclosure: I'm talking about parallel tempering as I learnt about it back in the day. `Pigeons` also has `AutoMala` https://proceedings.mlr.press/v238/biron-lattes24a.html I need to get to the bottom of whether that is an alternate sampler or blended into their meta-algo.
